robot/description/urdf/stand_link.urdf.xacro

```diff
@@ -7,7 +7,34 @@
     <!-- Stand link; this is a dummy link that is used to make camera placement and other sensors on the stand more convenient -->
     <!-- Stand link definition. Note: purposely left blank -->
     <link name="${tf_prefix}/stand_link">
+    </link> 
+
+    <!-- ZED Camera -->
+    <link name="${tf_prefix}/zed_camera_link">
+        <visual>
+            <origin xyz="0 0 0" rpy="0 0 0"/>
+            <!-- TODO: should be using the mesh but couldn't get it to work -->
```

Comment:
This is not a TODO since I believe no one was told to use the mesh for ZED stereo cam (remove comment)